tests/end2end/tests/test_end2end.py

```diff
@@ -132,46 +131,46 @@ def test_circuit_result_response(device: Device, api: API, numpy_circuit: Circui
     delete_job(api, job_id=result.job_id)
 
 
-@pytest.mark.parametrize("device", get_devices_listing_params())
-def test_vqa_posting(device: Device, api: API, vqa: VQA):
-    """Test whether a vqa can be sent to each device
-    Args:
-        api: api instance to call the server with
-    """
-
-    check_operation_possible_or_skip(Operation.POST, device)
-    job_id = api.execute(vqa=vqa, device_id=device.id)
-    assert isinstance(job_id, int)
-    api.delete_job(job_id=job_id)
-
-
-@pytest.mark.parametrize("device", get_devices_listing_params())
-def test_vqa_response(device: Device, api: API, numpy_circuit: Circuit):
-    """Test whether a vqa can be sent and then its result can be retrieved, for each device.
-
-    Args:
-        api: api instance to call the server with
-    """
-
-    logger = logging.getLogger(__name__)
-    logger.debug(f"device: {device}")
-
-    check_operation_possible_or_skip(Operation.POST, device)
-
-    result: JobData = post_and_get_result(api=api, device=device, circuit=numpy_circuit, timeout=15)
-    logger.debug(f"result: {result}")
-
-    # The operation post + response can be performed always (it is an async action) but
-    # the meaning of SUCCESS/EXCEPTION/FORBIDDEN means something different:
-    response_result = get_expected_operation_result(Operation.RESPONSE, device)
-    if response_result == OperationResult.SUCCESS:
-        assert result.status == JobStatus.COMPLETED
-    elif response_result == OperationResult.EXCEPTION:
-        assert result.status == JobStatus.ERROR
-    else:
-        assert result.status == JobStatus.PENDING  # offline device in legacy
-
-    delete_job(api, job_id=result.job_id)
+# @pytest.mark.parametrize("device", get_devices_listing_params())
```

Comment:
Why keep commented code? Either uncomment it, or remove it from the repo.

Reply:
Because technically we never took de official decision to definitely drop VQA support, we were kind of expecting "it" to be fixed... Should we bring out the scissors?

Reply:
I created the ticket for discussing this, though, because it is bigger than this specific comment: https://linear.app/qilimanjaro/issue/CLO-871/drop-vqa-code-from-qiboconnection-and-qgqs